src/Dev/DataExtensionAddsTrigger.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\StaticPublishQueue\Dev;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\StaticPublishQueue\Contract\StaticPublishingTrigger;
+use SilverStripe\StaticPublishQueue\Contract\StaticallyPublishable;
+
+class DataExtensionAddsTrigger extends DataExtension implements StaticallyPublishable, StaticPublishingTrigger, TestOnly
```

Comment:
```suggestion
class DataExtensionAddsTrigger extends Extension implements StaticallyPublishable, StaticPublishingTrigger, TestOnly
```
`DataExtension` is deprecated, we shouldn't be using it.